docs/getting-started/installation.md

```diff
@@ -42,7 +43,6 @@ $ bin/gremlin.sh
 gremlin> :remote connect tinkerpop.server conf/remote.yaml
 ==>Configured localhost/127.0.0.1:8182
 ```
-Notice that the client side of this works exactly the same as before when running both the client and server locally without Docker.
```

Comment:
This sentence can be misleading - not sure what "as before" means. Seems this section was after the local installation guide but now it's before the local installation guide. Anyways, I felt it was not important and thus removed it.